CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+
+#### Issues and Bug Reports
+
+If you have found an issue with goose3, please do not hesitate to let us
+know! Before submitting an issue or bug report, we ask that you complete a few
+cursory items:
+
+* **Review** current bugs to see if your issue has already been reported. If it
+has been previously reported, please comment on the original report with any
+additional details. This will help the maintainers triage the issue more
+quickly.
+
+* **Ensure** that the issue is **not** related to internet connectivity issues.
+Please ensure that static HTML files are also not working. These files of
+example issues will make finding the root cause of the defect easier.
+
+* **Determine** that the issue is reproducible - a code sample of the issue
+will help narrow down the search for the cause of the issue and may lead to a
+quicker fix! Also an example HTML file never hurts!
+
+A **great bug report** will consist of the following:
+
+* A descriptive title
+
+* A brief description of the issue
+
+* Description of the expected results
+
+* An code example to reproduce the error. Please use
+[Markdown code blocks](https://help.github.com/articles/creating-and-highlighting-code-blocks/)
+with syntax highlighting
+
+* The affected version(s) of goose3
+
+#### Enhancements
+
+Enhancements are additional functionality not currently supported by the
+goose3 library. Unfortunately, not all enhancements make sense for the
+goal of the project. If you have a desired feature, there are a few things you
+can do to possibly help get the feature into the goose3 library:
+
+* **Review** to see if the feature has been requested in the past.
+
+    * If it is requested and still open, add your comment as to why you would
+    like it.
+
+    * If it was previously requested but closed, you may be interested in why
+    it was closed and not implemented. I will try to explain my reasoning for
+    not supporting actions as much as possible.
+
+* Add an issue to the
+[issue tracker](https://github.com/barrust/mediawiki/issues) and mark it as an
+enhancement. A ***great enhancement*** request will have the following
+information:
+
+    * A descriptive title
+
+    * A description of the desired functionality: use cases, added benefit to
+    the library, etc.
+
+    * A code example, if necessary, to explain how the code would be used
+
+    * A description of the desired results
+
+#### Pull Requests
+
+Pull requests are how you will be able to add new features, fix bugs, or update
+documentation in the goose3 library. To create a pull request, you will
+first need to fork the repository, make all necessary changes and then create
+a pull request. There are a few guidelines for creating pull requests:
+
+* All pull requests must be based off of the latest development branch and not
+master (unless there is not a development branch!)
+
+* If the PR only changes documentation, please add `[ci skip]` to the commit
```

Comment:
Is this relevant?

Reply:
Not necessarily. I can easily remove it.